daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -1155,3 +1156,28 @@ class DaffodilUnparseContentHandler private[japi] (sContentHandler: SDaffodilUnp
   override def skippedEntity(name: String): Unit =
     contentHandler.skippedEntity(name)
 }
+
+/**
+ * Returns the EntityResolver used by Daffodil to resolve import/include
+ * schemaLocations.
+ *
+ * The entity resolver attempts to resolve namespaces and systemId's in the
+ * following order:
+ *
+ * 1. Use an org.apache.xml.resolver.Catalog/CatalogManager. By default the
+ *    Catalog only includes the daffodil-built-in-catalog.xml, but additional
+ *    catalogs can be added by putting CatalogManager.properties on the
+ *    classpath when daffodil is run.
+ *
+ * 2. If not resolved in step 1, schemaLocations are resolved relative to the
+ *    importing schema URI, which could either be a file on the filesystem or in
+ *    a jar on the classpath.
+ *
+ * The DFDLCatalogResolver isn't thread safe, but it also is expensive and stateful,
```

Comment:
One more instance of `DFDLCatalogResolver` to remove. New changes look good to me.